docs/UserGuide.adoc

```diff
@@ -287,35 +287,39 @@ For example, if you enter the wrong name and want to undo the action, type `/und
 .Form layout showing that the last entry is removed after you typed in `/undo`
 image::undoCommand.png[width="790"]
 
+// tag::list[]
 === Listing persons : `list`
+This list command can show you the list of persons in ORGANice.
 
-Shows a list of all doctors, donors, or patients in ORGANice, depending on the type of person specified. Shows all
-persons in ORGANice if no type is specified.
+==== Listing all persons
+Simply use `list` and ORGANice will show you all patients, doctors and donors present in the system.
 
 Format: `list`
 
-Optional parameters: Type parameter - `t/TYPE`
+==== Listing persons of a certain type
+If you wish to see a certain type of persons only, you can provide the type parameter in the list command.
```

Comment:
Suggestion: see a certain type of 'person'. (instead of 'persons')

Reply:
Amended